### PR TITLE
docs: create specific file for dockerhub

### LINKED ISSUE #122: Typo in Dockerhub repo description
Hi, i noticed a typo at the end of your Dockerhub repo description. (https://hub.docker.com/r/owasp/modsecurity-crs/)
In "Full docker run example of possible setup" there seems to be -p missing. It now looks like that:
`docker run -dti 80:80 --rm \`
and should look like that:
`docker run -dti -p 80:80 --rm \`

However this seems to not be the case on this github repo, where it's presented correctly.

### PR DESCRIPTION
**Pull request by fzipi**
Trimmed down version of the readme document.

Fixes #122.

### CODE REVIEW

**Comment by fzipi** [comment]
This version is still `30212 README-containers.md` bytes. We need to chop off 5k bytes more...

**Comment by fzipi** [comment]
@theseion One thing that I found while doing this is that we are using `/var/log/nginx/error.log` for the error log in nginx, but `'/proc/self/fd/2'` for apache. I think we should default bot to `'/proc/self/fd/2'`, wdyt?

**Comment by fzipi** [comment]
Also found https://github.com/coreruleset/coreruleset/issues/3144 while rewriting.

**Comment by fzipi** [comment]
Ok, needed to drop a lot but 
```
❯ wc -c README-containers.md
   24957 README-containers.md
```

**Comment by theseion** [comment]
> @theseion One thing that I found while doing this is that we are using /var/log/nginx/error.log for the error log in nginx, but '/proc/self/fd/2' for apache. I think we should default bot to '/proc/self/fd/2', wdyt?

I do agree. However, `/var/log/nginx/error.log` is actually a symlink to `/dev/stderr`, which is a symlink to `/proc/self/fd/2`. So, technically, we already do use the same default for nginx (which is in line with the documentation too).

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+name: Check lenght of dockerhub README file
```
Comment (theseion) [inline .github/workflows/lint.yml:2]:
```suggestion
name: Check length of Docker Hub README file
```

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+      - name: Check README-containers.md lenght
+        run: |
+          count=$(wc -m <README-containers.md)
+          if [[ $count -ge 25000 ]]; then
+            echo -n "README-containers.md contains $count characters"
```
Comment (theseion) [inline .github/workflows/lint.yml:23]:
```suggestion
            echo -n "README-containers.md contains ${count} characters"
```

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+      - name: Check README-containers.md lenght
+        run: |
+          count=$(wc -m <README-containers.md)
+          if [[ $count -ge 25000 ]]; then
```
Comment (theseion) [inline .github/workflows/lint.yml:22]:
```suggestion
          if [[ ${count} -ge 25000 ]]; then
```

File: README-containers.md [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+
+[![Build Status](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2Fcoreruleset%2Fmodsecurity-crs-docker%2Fbadge%3Fref%3Dmaster&style=flat)](https://actions-badge.atrox.dev/coreruleset/modsecurity-crs-docker/goto?ref=master
+) [![GitHub issues](https://img.shields.io/github/issues-raw/coreruleset/modsecurity-crs-docker.svg)](https://github.com/coreruleset/modsecurity-crs-docker/issues
+) [![GitHub PRs](https://img.shields.io/github/issues-pr-raw/coreruleset/modsecurity-crs-docker.svg)](https://github.com/coreruleset/modsecurity-crs-docker/pulls
+) [![License](https://img.shields.io/github/license/coreruleset/modsecurity-crs-docker.svg)](https://github.com/coreruleset/modsecurity-crs-docker/blob/master/LICENSE)
+
+## Full documentation
+
+⚠️ We are limited to 25000 chars in the dockerhub documentation. To see the full documentation please visit [our github README](https://github.com/coreruleset/modsecurity-crs-docker/blob/master/README.md). There you can find how to build the containers for multiple architectures, how to add your own CRS version and additional information.
```
Comment (theseion) [inline README-containers.md:12]:
```suggestion
⚠️ We are limited to 25000 chars in the Docker Hub documentation. The full documentation i hosted on [GitHub](https://github.com/coreruleset/modsecurity-crs-docker/blob/master/README.md). There you can find how to build the containers for multiple architectures, how to add your own CRS version, and additional information.
```

File: README-containers.md [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+## Supported architectures
+
+* linux/amd64
+* linux/i386
+* linux/arm64
+* linux/arm/v7
+
+## Quick reference
+
+* **Where to get help**: the [Core Rule Set Slack Channel](https://owasp-slack.herokuapp.com/) (#coreruleset on owasp.slack.com), or [Stack Overflow](https://stackoverflow.com/questions/tagged/mod-security)
```
Comment (theseion) [inline README-containers.md:45]:
```suggestion
* **Where to get help**: the [Core Rule Set Slack Channel](https://owasp.org/slack/invite) (#coreruleset on owasp.slack.com), or [Stack Overflow](https://stackoverflow.com/questions/tagged/mod-security)
```

**Review by theseion — CHANGES_REQUESTED** [review]
Thanks!

**Review by theseion — APPROVED** [review]

**Pull request merged**